src/packages/server/projects/control/kucalc.ts

```diff
@@ -55,7 +55,10 @@ class Project extends BaseProject {
     }
     try {
       this.stateChanging = { state: "starting" };
-      await this.siteLicenseHook();
+
+      // TODO: once "manage" processes site licenses, we can remove this line!
+      // Manage has do the equivalent of this.computeQuota()
+      await this.siteLicenseHook(false);
```

Comment:
I should add, `siteLicenseHook(false)` should stay there until the equivalent of https://github.com/sagemathinc/kucalc/pull/114 has been patched in cocalc-cloud